Source/RxCocoa/UIViewPropertyAnimator+Rx.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  UIViewPropertyAnimator+Rx.swift
+//  RxSwiftExt
+//
+//  Created by Wittemberg, Thibault on 29/03/18.
+//  Copyright © 2017 RxSwift Community. All rights reserved.
+//
+
+import Foundation
+import UIKit
+import RxSwift
+import RxCocoa
+
+@available(iOSApplicationExtension 10.0, *)
+extension Reactive where Base == UIViewPropertyAnimator {
+
+    /// Completable that when subscribed to, starts the animation
+    /// and completes once the animation is ended
+    public var animate: Completable {
+        return Completable.create(subscribe: { (completable) -> Disposable in
```

Comment:
What do you think about adding a `let strongBase = base` outside of the closure? I think that otherwise there's a risk of retaining self here.

Reply:
done but with a slightly different way. Until now I didn't notice we could use the capture list that way: 
```swift
{ [base] completable in ... }
```
What do you think of this syntax ?